app/controllers/merchant/items_controller.rb

```diff
@@ -3,6 +3,14 @@ def index
     @merchant = Merchant.find(current_user.merchant_id)
   end
 
+  def destroy
+    item = Item.find(params[:item_id])
+    Review.where(item_id: item.id).destroy_all
```

Comment:
We might be able to do a dependency to clean this up(remove line 8) but no need to worry about that now